styles/components/panel.scss

```diff
@@ -0,0 +1,48 @@
+@mixin panel-variant($bg, $color) {
+	border: 1px solid $bg;
```

Comment:
oso, i think its better to add the color of text in the arguments because tinatamaan si TABLE at LIST ng kulay white na font